Classes/Systems/Secrets.m

```diff
@@ -0,0 +1,27 @@
+//
+//  Secrets.m
+//  Freetime
+//
+//  Created by Ryan Nystrom on 11/19/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+#import "Secrets.h"
+
+#define STRINGIFY(x) @#x
```

Comment:
Not exactly sure what this is, but assuming this is environment variables as per your comment - could we use (in Swift) `ProcessInfo.processInfo.environment[x]` instead?

Reply:
Hmm maybe, just following #197 atm

Reply:
Mmm, alright I'm not too precious about it - just my lack of ObjC knowledge 😅

Reply:
Lol ya, same, its gonna be hacks no matter how we slice it. I'm def in uncharted territory atm lol.